src/Airport.js

```diff
@@ -0,0 +1,38 @@
+const Airplane = require(`./Airplane`);
+
+class Airport {
+
+    constructor() {
+    }
+
```

Comment:
If I were to do this again, I would keep the class properties and methods private and use setters and getters to connect the relevant information with the instantiated object.